Kitodo/src/main/java/org/goobi/production/constants/FileNames.java

```diff
@@ -17,10 +17,6 @@
  * @author Matthias Ronge &lt;matthias.ronge@zeutschel.de&gt;
  */
 public class FileNames {
-    /**
-     * Production main configuration file name.
-     */
-    public static final String CONFIG_FILE = "kitodo_config.properties";
```

Comment:
Instead of fixing usage of this constant value in `Kitodo-API/src/main/java/org/kitodo/config/Config.java` and `Kitodo-DataManagement/src/main/java/org/kitodo/config/ConfigMain.java`, you are removing it?

Reply:
Those two places don't see this attribute - both of those modules can't access Core module.

Reply:
So this file name must be maintained in many places. I don't like this solution but you must have good reasons to do it in this way.